packages/dumb/src/utils/datetime/datetime.util.ts

```diff
@@ -35,6 +35,8 @@ const t = i18next.t.bind(i18next)
 export const ONE_HOUR_MS = 3600000
 export const HOURS_IN_DAY = 24
 export const TIMEZONE_UTC = 'UTC'
+export const THIRTY_MINS = 1800000
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏What do you think of computing these values from `ONE_HOUR_MS` instead of having the plain value written here? If a mistake is made (e.g. removing a 0 at the end), personally I'm more likely to see it in `ONE_HOUR_MS` than in the other values (and in general, the less raw numbers we have in the code, the better)